modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js

```diff
@@ -481,7 +481,7 @@
 			// some initial calls to prime the seek:
 			if ( ( vid.currentTime === 0 && callbackCount === 0 ) && vid.readyState === 0 ) { //load video again if not loaded yet (vid.readyState === 0)
 				// when seeking turn off preload none and issue a load call.
-				if(mw.isIpad()){
+				if(mw.isIpad() || mw.isIOSAbove7() || mw.isIOS12()){
```

Comment:
We already have an example of getting safari above a specific version. It doesn’t make any sense to add ios12 like this.
Either patch the iOS above 7 to include 12 and anything above, or just change it to  isIos as I don’t believe there are too many devices that use ios6 and below if at all - and you can verify this in the Kava analytics dashboard.